src/main/cliapp/src/Login.js

```diff
@@ -9,7 +9,8 @@ export const Login = ({ children }) => {
 	const [, setCookie] = useCookies(['okta-token-cookie']);
 
 	const onSuccess = (tokens) => {
-		setCookie('okta-token-cookie', tokens.accessToken.accessToken);
+		//set cookie max age to 48 hours
+		setCookie('okta-token-cookie', tokens.accessToken.accessToken, { maxAge: 60 * 60 * 48 });
```

Comment:
@adamgibs do we know the token expire at okta?

Reply:
Ah, right. Do you know how to find that info? I'm not sure that I have access.